tools/infer/predict_system.py

```diff
@@ -65,15 +65,26 @@ def draw_crop_rec_res(self, output_dir, img_crop_list, rec_res):
         self.crop_image_res_index += bbox_num
 
     def __call__(self, img, cls=True):
-        time_dict = {'det': 0, 'rec': 0, 'csl': 0, 'all': 0}
+        time_dict = {'det': 0, 'rec': 0, 'cls': 0, 'all': 0}
+
+        if img is None:
+            logger.debug("no valid image provided")
+            time_dict['all'] = 0
```

Comment:
This line is redundant since time_dict only records the time elapse of this image, which is 0 as how it is initialized.